src/components/admin/form-components/ListSelector.jsx

```diff
@@ -41,9 +37,9 @@ export default class ListSelector extends React.Component {
 }
 
 ListSelector.propTypes = {
-  types: PropTypes.object.isRequired,
+  elements: PropTypes.object.isRequired,
```

Comment:
We could also specify here with `withShape` (or something along those lines) that the object needs 2 string fields, `slug`, and `name`